app/src/main/res/values/colors.xml

```diff
@@ -1,6 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <color name="colorPrimary">#3F51B5</color>
-    <color name="colorPrimaryDark">#303F9F</color>
-    <color name="colorAccent">#FF4081</color>
+    <color name="colorPrimary">#f75f52</color>
+    <color name="colorPrimaryDark">#f54d41</color>
+    <color name="colorAccent">#f75f52</color>
```

Comment:
primary and accent are set to the same value, be sure this is the intended affect.